html/pfappserver/root/authentication/source/type/HTTP.tt

```diff
@@ -0,0 +1,10 @@
+<div class="control-group">
+  <label class="control-label">[% l('Host') %]</label>
+  <div class="controls">
+    [% form.field('encryption').render_element | none %] [% form.field('host').render_element | none %]:[% form.field('port').render_element | none %]  
+  </div>
+</div>
+[% form.field('username').render | none %]
+[% form.field('password').render | none %]
+[% form.field('authentication_url').render | none %]
+[% form.field('authorization_url').render | none %]
```

Comment:
Will not work as the host configuration doesn't use default rendering and puts it in one line.

I could put the rest of the attributes in a definition but it wouldn't be reprensentative of the whole and could lead to confusion.

Unless there is a way to accomplish it using definition